java/test/jmri/InstanceManagerTest.java

```diff
@@ -145,6 +145,22 @@ public void testGenericStoreAndGetTwoDifferentTypes() {
 
     }
 
+    @Test
+    public void testListTwoDifferentTypes() {
+        PowerManager m1 = new PowerManagerScaffold();
+        PowerManager m2;
+        TurnoutManager t1 = new TurnoutManagerScaffold();
+        TurnoutManager t2;
```

Comment:
What's the purpose of m2 and t2? They aren't used.
I suppose static analysis will fail on this.

Reply:
Cut&paste error, now fixed.